### PR TITLE
added commentary column

### PR DESCRIPTION
**Pull request by brandonopened**
Josh please review - I added an extra column for commentary about how to use associations.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -2142,30 +2142,27 @@ OpenSALT allows for the associations defined by \ |LINK19|\ . The permitted asso
...
+    |exemplar        |The target/destination node is an example of best practice for the definition of the source/origin.                                                                                                                             |Point to a learning resource or other public artifacts(Unlimited).                |
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |hasSkillLevel   |The destination of this association is understood to define a given skill level i.e. Reading Lexile 100, Depth Knowledge 2, or Cognitive Level (Blooms Taxonomy) etc.                                                           |Used to capture equivalencies between academic and Workforce competencies.        |
```
Comment (roverwolf) [inline docs/index.rst]: I'm not sure this is quite correct.  Reading lexiles, DOK, etc. are not really workforce competencies but I think are more defining a "skill level" that is needed to perform the item.

Comment (brandonopened) [reply docs/index.rst]: I feel that's the one definition that includes relevant examples, true. 
"Used to capture equivalencies between different types of frameworks." ?

Comment (jmarks) [reply docs/index.rst]: That definition of hasSkillLevel, while correct, is too abstract to be helpful. I do like the examples. It is interesting, for the CASE best practices, that hasSkillLevel is similar to rubricCrtieriaLevel which is also a skill level. I am fine with the change with the suggestions.

Comment (jmarks) [reply docs/index.rst]: Shall I merge?

Comment (brandonopened) [reply docs/index.rst]: Please!

**Comment by brandonopened** [comment]
Please :-)

Open Standards, Open possibilities

________________________________
From: Joshua Marks <notifications@github.com>
Sent: Thursday, December 6, 2018 8:44:08 AM
To: opensalt/opensalt-docs
Cc: Brandon Dorman; Author
Subject: Re: [opensalt/opensalt-docs] added commentary column (#31)


@jmarks commented on this pull request.

________________________________

In docs/index.rst<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fopensalt%2Fopensalt-docs%2Fpull%2F31%23discussion_r239526248&data=02%7C01%7Cbrandon.dorman%40act.org%7C82cf328a5cbb4f6fd59408d65b9a0b20%7C65cb03469d8841d98ca6f72047670d0f%7C0%7C0%7C636797114498671185&sdata=nPkDd6lOfxvq8lzvKeutkqddBHVFqi8IhPxmVIpmJC4%3D&reserved=0>:

> -|isPeerOf        |The source/origin is a peer of of the target/destination.                                                                                                                                                                       |
-+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|isRelatedTo     |The origin of the association is related to the destination in some way that is not better described by another association type.                                                                                               |
-+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|precedes        |The origin of the association comes before the destination of the association in time or order.                                                                                                                                 |
-+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|replacedBy      |The origin of the association has been supplanted by, displaced by, or superseded by the destination of the association.                                                                                                        |
-+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |Association Type|Description                                                                                                                                                                                                                     |Commentary (Unofficial - not representative of official CASE documentation)       |
+    +================+================================================================================================================================================================================================================================+==================================================================================+
+    |exactMatchOf    |Equivalent to. Used to connect derived CFItem to CFItem in original source CFDocument.                                                                                                                                          |Used to connect derivative framework statements.                                  |
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |exemplar        |The target/destination node is an example of best practice for the definition of the source/origin.                                                                                                                             |Point to a learning resource or other public artifacts(Unlimited).                |
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |hasSkillLevel   |The destination of this association is understood to define a given skill level i.e. Reading Lexile 100, Depth Knowledge 2, or Cognitive Level (Blooms Taxonomy) etc.                                                           |Used to capture equivalencies between academic and Workforce competencies.        |


Shall I merge?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fopensalt%2Fopensalt-docs%2Fpull%2F31%23discussion_r239526248&data=02%7C01%7Cbrandon.dorman%40act.org%7C82cf328a5cbb4f6fd59408d65b9a0b20%7C65cb03469d8841d98ca6f72047670d0f%7C0%7C0%7C636797114498671185&sdata=nPkDd6lOfxvq8lzvKeutkqddBHVFqi8IhPxmVIpmJC4%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOQDr1CNACzQOgQnmk5ktq2zs428zSlvks5u2UlYgaJpZM4ZE_7g&data=02%7C01%7Cbrandon.dorman%40act.org%7C82cf328a5cbb4f6fd59408d65b9a0b20%7C65cb03469d8841d98ca6f72047670d0f%7C0%7C0%7C636797114498681193&sdata=5CkAjfJ746BkuM0sW0kWBfXm3DLLJWbwa37cet%2FUgpc%3D&reserved=0>.

**Pull request merged**